### PR TITLE
[SW-987] perf: use pre-generated ACIs instead of source code

### PR DESCRIPTION
**Pull request by CedrikNikita**

### CODE REVIEW

**Comment by Liubov-crypto** [comment]
LGTM

File: src/lib/contracts/FungibleTokenFullInterfaceACI.json
Comment (subhod-i) [inline src/lib/contracts/FungibleTokenFullInterfaceACI.json:1]: Only this file is sufficient, the Full ACI is not needed I believe.

File: src/popup/components/TransactionOverview.vue [hunk 1]
```diff
@@ -186,10 +188,10 @@ export default defineComponent({
...
       const sdk = await getSdk();
       const { bytecode } = await sdk.getContractByteCode(innerTx.value.contractId);
-      const txParams: ITx = await sdk.compilerApi.decodeCalldataBytecode({
-        bytecode,
-        calldata,
-      });
+      const txParams: ITx = await postJson(
+        `${activeNetwork.value.compilerUrl}/decode-calldata/bytecode`,
+        { body: { bytecode, calldata } },
+      );
```
Comment (subhod-i) [inline src/popup/components/TransactionOverview.vue:194]: Nice move. Add a todo to replace this with SDK method whenever we update the SDK version that supports this method.

**Review by subhod-i — APPROVED** [review]

File: src/lib/contracts/FungibleTokenFullInterfaceACI.json
Comment (CedrikNikita) [reply src/lib/contracts/FungibleTokenFullInterfaceACI.json:1]: `burn_trigger_pos` is not existing in this file, but required.

Comment (subhod-i) [reply src/lib/contracts/FungibleTokenFullInterfaceACI.json:1]: Then you could just use the just src/lib/contracts/FungibleTokenFullACI.json file only

Comment (CedrikNikita) [reply src/lib/contracts/FungibleTokenFullInterfaceACI.json:1]: `burn_trigger_pos` does not exist in `FungibleTokenFullACI`, but required by `ZeitToken`.

Comment (subhod-i) [reply src/lib/contracts/FungibleTokenFullInterfaceACI.json:1]: So maybe the file naming is misleading.

Comment (CedrikNikita) [reply src/lib/contracts/FungibleTokenFullInterfaceACI.json:1]: This is how the contract named inside of the file,

**Pull request merged**